Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -16,173 +16,94 @@ class MyTestSuite(EditorTestSuite):
        class MyFirstTest(EditorSingleTest):
            from . import script_to_be_run_by_editor as test_module
 
-       class MyTestInParallel_1(EditorParallelTest):
+       class MyTestInParallel_1(EditorBatchedTest):
```

Comment:
minor: Please revert this misleading change, which would not result in running two parallel tests.